integration/images/volume-ownership/Dockerfile_windows

```diff
@@ -25,10 +25,14 @@ WORKDIR C:/bin
 ADD tools/get_owner_windows.exe C:/bin/get_owner.exe
 RUN cmd.exe /c "@echo off && FOR %i in (%BUSYBOX_EXES%) do (mklink %i.exe busybox.exe)"
 
-RUN cmd.exe /c mkdir C:\volumes
-
 USER ContainerUser
 
+RUN mkdir C:\volumes
+# Explicitly set full access rights for "CREATOR OWNER". While
+# this is set by default for Windows Server 2019, it seems that
+# on Windows Server 2022 it is not.
+RUN icacls C:\volumes /grant "CREATOR OWNER":(OI)(CI)(IO)F /T
```

Comment:
(same here)

Also slightly wondering, but good for a follow-up; as the images are quite similar, perhaps we should combine the Dockerfiles (perhaps a separate stage for the `VOLUME`), so that they share the first couple of layers 🤔

Reply:
That is a good idea. I'm still learning the code base at this point. A common layer, or perhaps even a single image for volume tests may be best moving forward.

Reply:
Yes; especially for Windows images, which are known to be larger, and for which extra layers may involve quite some overhead.

anyway; not for this PR, just an observation / idea 🤗